.docker/ci-shadow-fixed/Dockerfile

```diff
@@ -1,79 +1,15 @@
 # moveit/moveit:melodic-ci-shadow-fixed
 # Sets up a base image to use for running Continuous Integration on Travis
 
-FROM ubuntu:18.04
+FROM moveit/moveit:melodic-ci
 MAINTAINER Dave Coleman dave@picknik.ai
 
-# TODO(davetcoleman): depend on ros:melodic-ros-base once OSRF makes
-# this available (currently melodic is not released yet)
-# https://github.com/osrf/docker_images/issues/134
+# Switch to ros-shadow-fixed
+RUN echo "deb http://packages.ros.org/ros-shadow-fixed/ubuntu `lsb_release -cs` main" | tee /etc/apt/sources.list.d/ros-latest.list
 
-# install packages
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    dirmngr \
-    gnupg2 \
-    && rm -rf /var/lib/apt/lists/*
+# Upgrade packages to ros-shadow-fixed
+RUN apt-get -qq update
+RUN apt-get -qq upgrade
 
-# setup keys
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
-
-# setup sources.list with shadow-fixed
-RUN echo "deb http://packages.ros.org/ros-shadow-fixed/ubuntu bionic main" > /etc/apt/sources.list.d/ros-latest.list
-
-# hack to fix issue with tzdata install
-# https://stackoverflow.com/questions/8671308/non-interactive-method-for-dpkg-reconfigure-tzdata
-ENV DEBIAN_FRONTEND=noninteractive
-ENV DEBCONF_NONINTERACTIVE_SEEN=true
-
-# install bootstrap tools
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-rosdep \
-    python-rosinstall \
-    python-vcstools \
-    && rm -rf /var/lib/apt/lists/*
-
-# setup environment
-ENV LANG C.UTF-8
-ENV LC_ALL C.UTF-8
-
```

Comment:
No need for this. `moveit/moveit:melodic-ci-shadow-fixed` derives from `moveit/moveit:melodic-ci` that already defines all this. Here we only need to update the packages to `ros-shadow-fixed`.

Reply:
the problem you're missing is that i spent a lot of time optimizing these docker images for the unified moveit repo so that travis doesn't time out. i have since then asked travis to give us extra time for free and they did, but im worries building CIs on top of each other too much will make them very large downloads. each RUN command is its own virtual machine layer, essentially